app/models/user.rb

```diff
@@ -27,15 +27,24 @@ class User < ApplicationRecord
 
   paginates_per 9
 
-  after_create :send_welcome_email_to_new_user
+  after_create :send_welcome_email_to_new_user, :send_mail_to_admin
 
   def send_welcome_email_to_new_user
     UserMailer.send_welcome_email(self).deliver_now
   end
 
+  def send_mail_to_admin
+    AdminMailer.admin_notification(self).deliver_now
+  end
+
   def full_name
     "#{first_name} #{last_name}"
   end
+
+  def self.admin
+    User.where(role: 0)
```

Comment:
```suggestion
    User.where(role: ADMIN)
```
так старайся не робити, бо не зрозуміло, що це за нуль
в тебе є константа для цього